### ISSUE TITLE
Menu Background and Mount Mayhem all white in Beetle Adventure Racing

### ISSUE THREAD

**Issue by loganmc10** [gonetz/GLideN64#1113]
![screen](https://cloud.githubusercontent.com/assets/848146/18407304/bd512efe-76c9-11e6-97ce-fa105e9b864b.png)

This is on Linux with Fedora 24 (Mesa 12.0.1 Intel Graphics).

It works when I compile with -DGLES2=On

**Comment by loganmc10** [comment]
It works for me on Windows, but not Fedora 24 (tested in VMware and on a laptop with Intel Graphics)

**Comment by gonetz** [comment]
Depth buffer fog requires support of image textures (GL 4.3) I doubt that VMWare drivers are so good to support that level of GL.

**Comment by loganmc10** [comment]
But why does it work with GLES2?

**Comment by gonetz** [comment]
Fog does not work with GLES2. It is ignored.
Probably your drivers declare image textures support, but do not properly implement it.
Also, there is a chance that image texture based functionality does not work on Linux. I need to test it.

**Comment by gonetz** [comment]
I tested current version on Linux workstation with NVidia card - Mount Mayhem level works properly.

**Comment by loganmc10** [comment]
@gonetz I've had a few other people test this and here is the issue.

OpenGL 3.3 + GL_IMAGE_TEXTURES_SUPPORT = broken.

If I take out ```#define GL_IMAGE_TEXTURES_SUPPORT``` in OpenGL.h then it works for me on my machine, which supports OpenGL 3.3

GL_IMAGE_TEXTURES_SUPPORT is defined anytime OpenGL is used, but it is only supported in OpenGL 4.3+

There is code that checks for Image Texture support (https://github.com/gonetz/GLideN64/blob/master/src/OpenGL.cpp#L1909-L1925) and it sets ```m_bImageTexture``` to true if it is, but that is not used very much, GL_IMAGE_TEXTURES_SUPPORT is used to check for Image Texure support.

Basically to fix this, the Image Texture code should be protected by ```if (m_bImageTexture)```, not ```#ifdef GL_IMAGE_TEXTURES_SUPPORT```

**Comment by loganmc10** [comment]
Actually I see that there is a isImageTexturesSupported() function, but it must not be used everywhere that it needs to, because removing the GL_IMAGE_TEXTURES_SUPPORT define fixes the issue on OpenGL 3.3
